bin/pre-flight-checks

```diff
@@ -83,6 +118,7 @@ check_envvars() {
         GITHUB_USERNAME
         HEROKU_API_KEY
         TRAVIS_KEYCHAIN_DIR
+        SLACK_WEBHOOK
```

Comment:
We should also get this one from the keychain automatically instead of requiring it to be copy pasted to `.env`.